### ISSUE TITLE
Apply screen-tearing fix on startup

### ISSUE THREAD

**Issue by maxdeviant** [maxdeviant/dotfiles#1]
Right now I have to run the following on startup each time to fix the screen-tearing issues with the graphics card:

```
nvidia-settings --assign CurrentMetaMode="DP-4:3840x2160_60 +0+0 { ForceFullCompositionPipeline = On }"
```

I should stop being lazy and put this into a config file or something so that I don't have to run it all the time.

**Comment by maxdeviant** [comment]
I partially opened this issue just so I have a reference in case my shell history every gets wiped :sweat_smile:

**Comment by maxdeviant** [comment]
Screen tearing fix for the new dual monitor setup:

```
nvidia-settings --assign CurrentMetaMode="DP-2: nvidia-auto-select +0+0 { ForceFullCompositionPipeline = On }, DP-4: nvidia-auto-select +3840+0 { ForceFullCompositionPipeline = On }"
```

For more info, see the [Arch Linux wiki](https://wiki.archlinux.org/index.php/NVIDIA/Troubleshooting#Avoid_screen_tearing)

**Comment by maxdeviant** [comment]
Fixed in 6caee6df053a208c8eb8feb15df2a50beabaf2a8.
